infrastructure/subscription_level/main.bicep

```diff
@@ -0,0 +1,14 @@
+param location string = 'uksouth'
+param resourceGroupName string = 'Digital-Feature'
+
+
+// I probably don't need a resource group module and can probably do everything here...
```

Comment:
Note that the `infrastrcuture.yml` infrastructure can also be done in Bicep. (I.e. the stuff that isn't scoped to the Resource Group.) Note that these are scoped at the subscription level.
I've kept it separated for now, but it may not be necessary.